Automattic/vip-go-mu-plugins
# Use unique file name provided by VIP files service.

## Description
This PR Changes the filename of uploads to be the unique value provided by the VIP files service.  The file type validation was updated to also use the unique file name since it is a part of the same API call.

## Changelog Description

### Use Unique Filename from Files Service
Update filename for uploads to match unique value from VIP Files Service

## Pre-review checklist

Please make sure the items below have been covered before requesting a review:

- [x ] This change works and has been tested locally (or has an appropriate fallback).
- [ x] This change works and has been tested on a Go sandbox.
- [x ] This change has relevant unit tests (if applicable).
- [ n/a ] This change uses a rollout method to ease with deployment (if applicable - especially for large scale actions that require writes).
- [x ] I've created a changelog description that aligns with the provided examples.

## Pre-deploy checklist


## Steps to Test
Test in Sandbox

1. Check out PR.
2. Go to media library
3. Upload file
4. Upload file again.
5. Verify that first file is saved with original filename and second filename has unique 6 char hash added to original filename.
